WordPress-VIP/ruleset.xml

```diff
@@ -211,6 +211,16 @@
 		<severity>0</severity>
 	</rule>
 
+	<!-- Prevent deprecation notice when the sniff is not explicitely included. -->
+	<rule ref="WordPress.VIP.FileSystemWritesDisallow.DeprecatedSniff">
+		<severity>0</severity>
+	</rule>
+
+	<!-- Prevent duplicate messages from deprecated sniff. -->
+	<rule ref="WordPress.VIP.FileSystemWritesDisallow">
```

Comment:
This would disable the sniff completely in contrast to deprecating it. I think this block should be removed.